### PR TITLE
Inserted more records

### PR DESCRIPTION
**Pull request by lena-b1**

### CODE REVIEW

**Comment by vldzz** [comment]
Good job!

File: Coffee_Time.sql [hunk 1]
```diff
@@ -93,62 +95,150 @@ GO
 
 ------------------------------------------------------[INSERT SECTION]----------------------------------------------------------
 
-
 INSERT INTO Subsidiaries VALUES
 	(1, 'Centru', 'str.Stefan cel Mare 47'),
-	(2, 'Botanica', 'str.Cuza Voda 128'),
-    (3, 'Ciocana', 'str.Mircea cel Batran 26')
```
Comment (Alexenon) [inline Coffee_Time.sql:100]: De ce ai scos Ciocana ?

File: Coffee_Time.sql [hunk 2]
```diff
@@ -1,3 +1,5 @@
+
+
```
Comment (Alexenon) [inline Coffee_Time.sql:2]: Pentru spatiile astea ?

**Review by Alexenon — COMMENTED** [review]
Review my comments and edit once more

File: Coffee_Time.sql (hunk 1 above)
Comment (lena-b1) [reply Coffee_Time.sql:100]: Ciocana nu merita filiala

File: Coffee_Time.sql (hunk 2 above)
Comment (lena-b1) [reply Coffee_Time.sql:2]: Buna intrebare

File: Coffee_Time.sql (hunk 1 above)
Comment (Alexenon) [reply Coffee_Time.sql:100]: La fel cum tu loc de munca, dar lasam detaliile

**Review by vldzz — COMMENTED** [review]
Good job, but not enough

**Pull request closed without merging**